apps/integration_tests/selenium_cases.py

```diff
@@ -23,6 +23,7 @@ class Action(Enum):
     GET_SAMPLE_TOKEN_PKCE_START = 11
     SLEEP = 12
     VALIDATE_EMAIL_NOTIFICATION = 13
+    CHECK_DATE_FORMAT = 14

```

Comment:
I've also never used selenium before. Can you briefly explain how the selenium_cases relates to the other files you touched/created ?

Reply:
Sure, Selenium runs automated web application test cases with a web driver and the actions in each test case are defined in this cases file. It contains things like id's of the HTML elements so the webdriver knows what element to click or what elements to search for, for validating what you want to validate. So this PR included:
1. Adding identifiers to HTML elements on the page that I want to test, in this case the authorization screen, so that Selenium can find them.
2. Adding and defining Selenium actions like click english, click spanish, check text etc for the test cases.
3. Defining what the checks should do, including the expected values of text etc.

I thought it best to put these new cases in a separate file from existing because we use the Selenium tests as a smoke test after any kind of deployment and I didn't want the existing ones to be affected by whether or not these changes were merged yet or not. The existing script can be amended locally to run it if you want as described in the earlier comment.

Reply:
Thank you for explaining Selenium to me! This PR is more clear to me now 🎉